gmso/formats/lammpsdata.py

```diff
@@ -71,12 +68,12 @@ def write_lammpsdata(topology, filename, atom_style='full'):
             str(datetime.datetime.now())))
         data.write('{:d} atoms\n'.format(len(topology.sites)))
         if atom_style in ['full', 'molecular']:
-            if bonds != 0:
-                data.write('{:} bonds\n'.format(len(bonds)))
+            if len(topology.bonds) != 0:
+                data.write('{:} bonds\n'.format(len(topology.bonds)))
```

Comment:
I'm just curious what purpose the colon serves here? I am unfamiliar with this syntax

Reply:
I think that's a typo.